src/Arcus.Observability.Tests.Unit/Telemetry/Logging/HttpDependencyLoggingTests.cs

```diff
@@ -417,7 +417,7 @@ public void LogHttpDependencyWithRequestWithDurationMeasurementWithDependencyId_
             // Arrange
             var logger = new TestLogger();
             HttpRequest request = CreateStubRequest(HttpMethod.Get, "host", "/path", "http");
-            var statusCode = BogusGenerator.PickRandom<HttpStatusCode>();
+            var statusCode = (HttpStatusCode)BogusGenerator.Random.Int(100, 599);
```

Comment:
Good thing that these tests are adapted so that we can be sure that the guard for `HttpStatusCode` is removed. 👍